### DISCUSSION TITLE
"url" parameter is valid but upstream response is invalid

### DISCUSSION THREAD

**Discussion by Iulian-Dragomirescu** [vercel/next.js#56037] [Help]
### Summary

HI, 
Image works in development and production (only locally, not remotely), but in nginx I get this error (from nginx console):
```
upstream image response failed for /assets/icons/gdpr.png TypeError: fetch failed
cause: ConnectTimeoutError: Connect Timeout Error
```

my folders
public/assets/icons/gdpr.png

```
        <Image
          onError={(e) => {
            console.error("Error loading image:", e);
          }}
          src="/assets/icons/gdpr.png"
          className="rounded-sm"
          alt="gdpr"
          width={100}
          height={100}
        />
```

i try to convert the image in webp.. same error
restart server.. same

and this: 
```"url" parameter is valid but upstream response is invalid```
i receive from chrome ```exemple.com/_next/image?url=%2Fassets%2Ficons%2Fgdpr.png&w=256&q=75```

<img width="265" alt="image" src="https://github.com/vercel/next.js/assets/54176220/5dfa8d65-6d90-4c65-8355-7da4ed560334">

Any ideea? 

Thanks!

### Additional information

```js
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 10 Home
Binaries:
  Node: 18.17.1
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 13.5.3
  eslint-config-next: N/A
  react: 18.2.0
  react-dom: 18.2.0
  typescript: N/A
Next.js Config:
  output: N/A
```

**Comment by mike-petrov** [comment]
The same error, see: https://github.com/vercel/next.js/issues/56038

**Comment by tlchatt** [comment]
I'm getting the same error did you static import your image? I am unable to do so because my file names come from props loaded in via fetched data. It seems to me to be a problem with my nginx server config. Works fine on dev and start if not via a domain.
